### DISCUSSION TITLE
Is doubleshock 4 controller supported? (Not the official ps4 dualshock)

### DISCUSSION THREAD

**Discussion by konstantinos-theod** [ndeadly/MissionControl#575] [Q&A]
What the title says. Careful I am referring to the 3rd party controller ps4 alike controller not the official one.

I am thinking of buying one and I don't want it to go to waste

https://www.skroutz.gr/s/23303339/Doubleshock-%CE%91%CF%83%CF%8D%CF%81%CE%BC%CE%B1%CF%84%CE%BF-Gamepad-%CE%B3%CE%B9%CE%B1-PS4-Camouflage-Blue.html?product_id=141809414

**Comment by ndeadly** [comment]
In theory, yes, but there have been accounts of certain knock-off controllers using incompatible hardware. It will probably work, but you're buying at your own risk.

**Comment by konstantinos-theod** [comment]
Can anyone that has a doubleshock with the normal/original x [] o /\ characters confirm? Like in this picture https://www.skroutz.gr/s/23303339/Doubleshock-%CE%91%CF%83%CF%8D%CF%81%CE%BC%CE%B1%CF%84%CE%BF-Gamepad-%CE%B3%CE%B9%CE%B1-PS4-Camouflage-Blue.html?product_id=141809414

**Reply by REXEL-OWO** [reply]
are you buy the doubleshock? im buy a generic controller ps4 but dont work :c
